test/tensors/test_native.py

```diff
@@ -0,0 +1,63 @@
+from syft.frameworks.torch.hook import TorchHook
+from syft.frameworks.torch.tensors import TorchTensor
+from syft.frameworks.torch.tensors import PointerTensor
+from syft.workers.virtual import VirtualWorker
+from unittest import TestCase
+import torch
+
+
+class TestTorchTensor(TestCase):
+    def test_owner_default(self):
+        hook = TorchHook()
+        tensor = TorchTensor()
```

Comment:
this should probably be replaced with a test which initializes a torch tensor the normal way.

x = torch.Tensor([1,2,3,4,5])

or

x = torch.zeros(5)

something like that.